project/frontend/src/pages/HomePage.js

```diff
@@ -84,6 +103,20 @@ function HomePage(props) {
       bottom='64px'>
       <PreferenceForm history={props.history} />
     </Modal>,
+    <Modal
+      key='location-form'
+      open={isLocationFormOpen}
+      onDismiss={toggleLocationForm}
+      centerHorizontal={true}
+      top='64px'
```

Comment:
I asked this in our chat but is having a set px dynamic for different screen sizes?

Reply:
I don't think it is-- agreed that making it dynamic would better. I just did this to make it the same as the other modal but we can discuss it in our meeting